lib/std/Build/Cache/DepTokenizer.zig

```diff
@@ -201,7 +211,7 @@ pub fn next(self: *Tokenizer) ?Token {
                 '\n' => {
                     self.index += 1;
                     self.state = .rhs;
-                    return Token{ .prereq = self.bytes[start .. self.index - 1] };
+                    return finishPrereq(must_resolve, self.bytes[start .. self.index - 3]);
```

Comment:
Note the offset here is intentionally changed from -1 to -3; this was a bug.

See test added at L586